core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/udf/r/RUDFSourceOpDesc.java

```diff
@@ -0,0 +1,117 @@
+package edu.uci.ics.texera.workflow.operators.udf.r;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import com.google.common.base.Preconditions;
+import com.kjetland.jackson.jsonSchema.annotations.JsonSchemaTitle;
+import edu.uci.ics.amber.engine.architecture.deploysemantics.PhysicalOp;
+import edu.uci.ics.amber.engine.architecture.deploysemantics.SchemaPropagationFunc;
+import edu.uci.ics.amber.engine.architecture.deploysemantics.layer.OpExecInitInfo;
+import edu.uci.ics.amber.engine.common.AmberUtils;
+import edu.uci.ics.amber.engine.common.virtualidentity.ExecutionIdentity;
+import edu.uci.ics.amber.engine.common.virtualidentity.WorkflowIdentity;
+import edu.uci.ics.amber.engine.common.workflow.InputPort;
+import edu.uci.ics.amber.engine.common.workflow.OutputPort;
+import edu.uci.ics.amber.engine.common.workflow.PortIdentity;
+import edu.uci.ics.texera.workflow.common.metadata.OperatorGroupConstants;
+import edu.uci.ics.texera.workflow.common.metadata.OperatorInfo;
+import edu.uci.ics.texera.workflow.common.operators.source.SourceOperatorDescriptor;
+import edu.uci.ics.texera.workflow.common.tuple.schema.Attribute;
+import edu.uci.ics.texera.workflow.common.tuple.schema.Schema;
+import scala.Option;
+import scala.collection.immutable.Map;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+
+import static java.util.Collections.singletonList;
+import static scala.jdk.javaapi.CollectionConverters.asScala;
+
+
+public class RUDFSourceOpDesc extends SourceOperatorDescriptor {
+    @JsonProperty(
+        required = true,
+        defaultValue =
+            "# Input: None\n" +
```

Comment:
I suggest remove this comment if possible. Better to use some annotation for implying the output type